docs/logical_data_model.encoded

```diff
@@ -1 +1,2 @@
sxR5f0oHhlObO7-xDiUcFFaX5AtAncKmtu0_ZfDtWLcUslGKdkcmJuWv7XmNIdmQT-d35Q-lJUM5ZCMPefTAwfSvKlazHgmpXgdNL75Bgs3e8PHcP02uus6s0us3TUO1PfZKok3disSlN_BVu5Z3eA7oWKIRhlYdCloDvUeoOQ6G4Tc9funfZr-PGrYWoMMoP6rmZ3JATkQ4opmVyXKD7ihOKv7GOYmq1f5djfGKFEoTeuo5YeqRq1qYqTu2bhhT0zWYFQEgAFamRBYWNGUpJQrJvqPQbRQ9XiRDuZOBm_FnkyWG74XINQQejgJT8vmx52SYUDp8B9DNV9OvXf6MLCrjewHIhZtCFRuZ-DxYIctOsJtyZkPuf6idp_NnTbY04v-HsBYRFfQLTeM5YQRaeXugE9Oq9EaogHgD9uJ73CGing_llOVBgiLttThUo3Mk3Pw03nTYPZLckuloOykhydgM8BHCqCRWsKit3XO8GS8YsLUERYBpSpbvRfX1ioX0B003xOYNtLHOtV2cH-Fk2wqFs-Wnpfwf9mmVuA7I-GST38EJz7dtAvocO8w_lyECaS1ga7SepF0-pyqXeywgCVHqoLkrOs0rmQAJRSvzW6zHTVsytfz5pqVnNPaoB7FVB4QJcdoub7x2pUCwJqYsaFnWsg2XqJ_yRI8UasA6T0NZlXHlGQ1pSzeLDgO7xJ-cmEE9PtGZNJABziMOZGFabQiC_5acmc-6L8tR46sgIotuPtLvkVh-nu_5yI60ln6jZdZxBTtXor6Mvm-hrxdjI9cDgL8sBivMXH8OH2dkfvAYTUiZDJuDirrqtXLa-65rrlKAehPy6MvatIWjHDakl4crvmF2u8RH-nlT8elmlQ4qxrugG_VYuctN3qtklmYHienlihdT2XR1fhHSAi1lzA0KxusIYg0PdvPN33cGwY2rl3d80LK7gkRKEO3g07KzWkVGCg5b29Cwe4n0EO0w06fB17H8jIBb5juGjEhH5WTLArGEfGwKtAgp0Cc1eu7JySeJ1y_gxFNhipv0VBhReIYuIT6c4NUGALatnYXTuc_n3jqO0pbvFn9WaOfb2am6jeZnbW8lvTb3qNiiO2e-7M-9-MEfZwe0KpLA1SZPZ8qrvVpOVdpSKTuAJVTBO1nAEafTpqiuh7IitABhXSX1qu36GQ93KdiZWCE4Os9uLO839WV6OQ9XCkHT3NybN3hUoE3WgU1ni6YmkCiR2spSakh6f-Mb30uZ0qPEb-nmSfT-5Y0vV5hmSShJ3ex3HSV5_4I-WsPrPdp-tm43m3GU_NUehCvSLdWRBXktX9gWE3lO18lXKNw7YyJ5GQ93OdmnW4DcTLP_gjzX0fZqlIN0vIEPON8GrI4ol6x0hR3LcJkscvWONYtWO96mjB-Nm26a8UIGQI3J_YO4Xv14o46YGM9-su13XYDJVtl2NnYLnJDlbIJK9_B3Z53GyR2-T_J5aVVYnO-VhKFaVre6ADdYO9bkYVyf0Zd-_UN75oz-yxoZwAZvuM3WwjVBbXtVzbqOdiaTmB_lGN4G1pMQB0rbUhVPfIg3OjO2bJqMRKy4ZXP8JmrfI5HMBzigHByzojMELhziZu7RTV4Lzg72oOuroSO9L4Ja6r8icn--duUZzkOJ4zkYRZPk-gxKDT16wzbaCA-HfSUgsjbulSrVwzazJj5gDfxAkkXZN3MTZNlxIgypRrlNGtYNgOB0BQa-ebUKBgitb2fbkXYBneD_0_WROb1S4hKK68gNHYwatatLMlx2JLrNfz5j71qgWzhvC9HJWpRL2fQ2rtYgMYwRIbsh6Ccb9GXRpQvAlErXhddj7lbTxs9kAjDc1kuz_4oSXf8tIlXI-upfCcSv3wDc2bQ3tQQp-InIgFjXE_DgBvp2i-2HwCGNqwtTWhBvv1fAtwXjahokQqE9ri6MEEZWZjfgjEqbXtSX7uEPQkk9Jp4OJRp1zCkPaYGtSd86Kv4TNfPVK-NZvN_7a2gV5X9YbRz6KopIUUJs9FwjQF4dSAP9jrI5N5JMnN2BH5N5uZENo_A_IYKDkzICsrtZIEpTyoR9OtyqBrjEwvdvgxDW8yDCyzAI8cqfHEvJpYD1VUTgS5lCwKTht3gyZL4ysyt9PSsod4ZVg3KA3uEMZDEEXbQeyvDhWPhpElMiKRw9PJnnnZBJvrzbKgdUGf5an65KEdBvTZ-Idfg9mwagFnLdSP7d0OvU459YJgE-298Hj7MRiAAiRkMZw5RWvWGBP8TICXu-1LNQF6M6Sr0Id1NISLKwWcQFCd1o8wYAEoqUv-T_byfxfnVLlZ9NvBdu0cjtl7awN-FMsCRS4BcTBe4Dusdysh7TJ4QppIFKUJIja36yJ1PenQTfWOxXqQCFT1uDw3Jd3DmC-D5XKH_JxHV_biMoUGSsc7QLBrmIPMNHO6fbkaw-q-JtSfpYGYPLZjvSnHSMzRtL5EUSCPPM18ZLqDeYnsg-kntnIRE-cvp1NDjj_PnVctq_WLChXv6KDTNCYIgwgMRu35JHUkgKc7DHj5G_brBMB-JfESTj5YTV14lByedqw_TNAW_zjceQ7t5x8Zv_iPGVsPRVudZrK30RpY0FZnGoYpZ7S2hwJY59gHl5K7er4op7KoaSJB7AUUMP2_oR9np4f2mTJB2-qowh_NOKBSEIgCXENW-cdBEImrsjeYTf1CZzhd2bQMhWMkmVnQcMw6TbI-Q5FMd3aP_ONA2ghharD_KIB6dSWit2mcf0yxBKj_Pao2tmDEIc4AnTQDIzw0c4guYl34NFHxebwlO_0Zjv9hLqMIjl5CugZnRFKKCpP5v6LafTNY0FnxCpfftVmauWNm-IZYFNcGnGKcs_hHsxD5ekJQUXhCUMnf9kqOicyPc8LjTT0xQ3t2wI5b4P15CkVwPALlZSvfZ-LEhKxX9d_6dHnM1E-KUIRPTNYGdQBsBfE-TUS1a7B1tqyM8nDMfWJQOMJPH42WT_B1KvjiNY6nm5sIvS_1y0
```

Comment:
I don't see a migration file did you mean to change these files.

Reply:
Might be the line endings from my checkin. 

I just switched it so hopefully its fixed.

Reply:
> I don't see a migration file did you mean to change these files.

I had a merge conflict even though this PR doesn't introduce any DB changes. I resolved the conflict by accepting changes from `main`. I may have not grabbed a line ending or something, but the hash itself is likely the same as what's on main

(we need better guidance around this stuff)